apps/root/src/pages/earn/components/faq/index.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import {
+  Typography,
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  ContainerBox,
+  colors,
+  ThumbsSatisfaction,
+} from 'ui-library';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import useTrackEvent from '@hooks/useTrackEvent';
+
+const StyledQuestion = styled(Typography).attrs({ variant: 'h6' })``;
+
+const StyledAnswer = styled(Typography).attrs({ variant: 'bodyRegular' })`
+  ${({ theme: { palette } }) => `
+  color: ${colors[palette.mode].typography.typo3};
+`}
+`;
+
+const StyledContainer = styled(ContainerBox)`
+  max-width: 630px;
+`;
+
+const EarnFAQ = () => {
+  const trackEvent = useTrackEvent();
+
+  const onFeedbackClick = ({ id, value: helpfull }: { id: string; value: boolean }) => {
+    trackEvent(`FAQ ${id} was usefull`, { helpfull });
+  };
+
+  return (
+    <StyledContainer flexDirection="column" gap={8} alignSelf="center">
+      <Typography variant="h4" fontWeight={700} textAlign="center">
+        <FormattedMessage description="earnFaqTitle" defaultMessage="Balmy’s Earn Frequently Asked Questions" />
+      </Typography>
+      <ContainerBox flexDirection="column">
+        <Accordion>
+          <AccordionSummary>
+            <StyledQuestion>
+              <FormattedMessage
+                description="faqWhatIsMeanFinance"
+                defaultMessage="What is Dollar Cost Averaging (DCA)?"
+              />
+            </StyledQuestion>
+          </AccordionSummary>
+          <AccordionDetails>
+            <StyledAnswer>
+              <FormattedMessage
```

Comment:
Lets leave only one of this and with a diferent description in case we change it on the original one so it does not impact translations